docs/pages/development.js

```diff
@@ -243,6 +243,63 @@ yarn cypress open
 ~~~"
                   />
                 </li>
+                <li>
+                  <Text>
+                    Run{' '}
+                    <Link href="https://github.com/meinaart/cypress-plugin-snapshots" inline>
+                      <Text weight="bold">Cypress snapshot tests</Text>
```

Comment:
Given the use of "snapshot" in Jest, we should clarify that these are visual diff tests. Something like "Cypress visual diff snapshot tests"?

Reply:
DONE